app/grandchallenge/algorithms/views.py

```diff
@@ -1013,3 +1022,52 @@ def get_success_message(self, cleaned_data):
 
     def get_success_url(self):
         return self.algorithm.get_absolute_url() + "#models"
+
+
+class AlgorithmImageTemplate(ObjectPermissionRequiredMixin, DetailView):
+    model = Algorithm
+    permission_required = "algorithms.change_algorithm"
+    raise_exception = True
+    queryset = Algorithm.objects.prefetch_related("inputs", "outputs")
+
+    def get(self, *_, **__):
+        algorithm = self.get_object()
+
+        forge_context = get_forge_algorithm_template_context(algorithm)
+
+        # Step 1: Create a temporary directory for generating content
```

Comment:
The LLM comments can be removed. L1039 tells a developer that a temporary directory is being generated. If they don't understand it then they can ask an LLM to explain it.